### PR TITLE
SP city center scenario e dataframes

### PR DESCRIPTION
**Pull request by lucasccdias**

### CODE REVIEW

**Comment by Robinlovelace** [comment]
Thanks for the updated PR for SP city centre. Testing locally after running these commands in the terminal:

```bash
git checkout -b lucasccdias-main main
git pull git://github.com/lucasccdias/abstr.git main
```

**Comment by Robinlovelace** [comment]
Looks good, are the entities in `ID_PESS` agents?

```
> library(dplyr)
> sp_min = sao_paulo_activity_df_20 %>% 
+   select(CO_O_X, CO_D_Y, ID_PESS, MODO1, H_SAIDA, MIN_SAIDA)
> table(as.character(sp_min$ID_PESS))

00010027105 00020646102 00032647101 00040018101 00042040104 00080506103 00090616106 00180877103 00241431103 00241455101 
          4           3           3           2           2           2           1           4           2           3 
00241517103 00250009103 00780788101 00891431101 01280562102 01891406103 02670034102 03291600102 03401483101 04551113101 
          4           2           2           2           2           1           2           2           1           2 
```

**Comment by lucasccdias** [comment]
Yes, `ID_PESS` is the person identifier in the survey.

**Review by Robinlovelace — APPROVED** [review]

File: data-raw/sao-paulo-activity-data.R [hunk 1]
```diff
@@ -9,7 +9,7 @@ unlink("OD-2017", recursive = T, force = T)
 
 # just São Paulo municipality with short trips
 OD_SP_2017 = OD_SP_2017 %>%
-  filter(MUNI_DOM == 36 & DISTANCIA <= 5000)
+  filter(ZONA_O %in% c(1:9, 18:19, 24:26) & ZONA_D %in% c(1:9, 18:19, 24:26))
```
Comment (Robinlovelace) [inline data-raw/sao-paulo-activity-data.R:12]: I think we should include a version with fewer columns. I can add that post merge.

**Review by Robinlovelace — APPROVED** [review]

**Pull request merged**